src/test/java/tests/FilterParserTests.java

```diff
@@ -123,6 +147,23 @@ public void operators() {
                     new Qualifier(LABEL, "urgent")));
         assertEquals(Parser.parse("milestone:0.4 state:open OR label:urgent"),
             Parser.parse("milestone:0.4 AND state:open OR label:urgent"));
+
+        try {
+            Parser.parse(";");
+            fail("Operator by itself should cause a parse exception.");
+        } catch (ParseException ignored) {}
+
+        try {
+            Parser.parse("&&");
+            fail("Operator by itself should cause a parse exception.");
+        } catch (ParseException ignored) {}
+
+        //might move this test in the future when proper
+        //semantic exception is implemented
```

Comment:
In that case, you can prefix the comment with "TODO" and include the number of the issue it's referring to.